ocs_ci/deployment/vmware.py

```diff
@@ -233,6 +233,23 @@ def modify_scaleup_repo(self):
         # change root disk size
         change_vm_root_disk_size(constants.SCALEUP_VSPHERE_MACHINE_CONF)
 
+    def post_destroy_checks(self, pool, dc, cluster):
```

Comment:
Could be done for pre check as well ? This can help cleanup existing pool if its somehow missed by manual cleanup.

Reply:
problem with pre-check is if someone has same cluster name, it will delete the cluster